jwst/tweakreg/tweakreg_step.py

```diff
@@ -136,20 +179,9 @@ def _imodel2wcsim(self, image_model):
                         'v2_ref': refang['v2_ref'] / 3600.0,
                         'v3_ref': refang['v3_ref'] / 3600.0},
             catalog=catalog,
-            name=_get_filename(image_model),
+            name=image_model.meta.filename,
```

Comment:
The reason for `_get_filename` was to reduce the length of the file name so that log messages are not excessively long. I am fine with keeping full path if that is what you really want.

Reply:
Thanks Mihai.  The reason I removed this is that I think the filename complete path will eventually be handled by stpipe, not individual steps.  Only the filename (not the absolute directory) will get written to this keyword, so we shouldn't have to do any special handling at the step level.  Is this a correct assumption @stscieisenhamer?